src/System.Private.Xml/src/System/Xml/XPath/Internal/XPathArrayIterator.cs

```diff
@@ -85,6 +85,6 @@ public override IEnumerator GetEnumerator()
             return list.GetEnumerator();
         }
 
-        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
+//        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
```

Comment:
Found similar issues when I deleted 18.5k lines of Roslyn which is why I flagged the branch as for review only. The other issues with Roslyn were regression (code should have been called), partial implementation (specifically of tests), non-conditional compilation.